src/BloomExe/Spreadsheet/SpreadsheetExporter.cs

```diff
@@ -75,7 +75,17 @@ public SpreadsheetExporter(ILanguageDisplayNameResolver langDisplayNameResolver)
 		{
 			var spreadsheet = ExportToFolder(dom, bookFolderPath, outputFolder, out string outputFilePath,
 				progress);
-			resultCallback(outputFilePath);
+			// ExportToFolder will return null if export fails, in which case we don't want to open/foreground the spreadsheet
+			if (spreadsheet != null)
+			{
+				resultCallback(outputFilePath);
+			}
+			else if (!progress.HaveProblemsBeenReported)
```

Comment:
Not clear why you don't want to say it failed if previously we said something specific about what went wrong?

Reply:
I really meant this as more of a stopgap. I'm not even sure what to put after the colon in "Export failed:". I think the existing messages about things that went wrong are pretty clear on their own. Otherwise we will get something like this:
![image](https://github.com/BloomBooks/BloomDesktop/assets/22460975/9812fc2a-47d0-4e52-94df-1c1cd4f8172a)